src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -137,6 +152,74 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
     props.onClose()
   }
 
+  const isTimeInFuture = timeToCheck => {
+    return moment(timeToCheck).isAfter(moment())
+  }
+
+  const validateStartTime = () => {
+    if (isTimeInFuture(startTime)) {
+      return {isValid: false, message: START_TIME_IN_FUTURE_MESSAGE}
+    }
+    return {isValid: true}
+  }
+
+  /**
+   * If there is a problem with the end time (with respect to the start time)
+   * return an error message here, along with the isValid flag
+   *
+   * If no message, then it is valid (just return a true isValid flag)
+   *
+   * since this is only vaild for range annotations, if it is a point annotations just return true
+   * */
+  const validateEndTime = () => {
+    if (annotationType === 'point') {
+      return {isValid: true}
+    }
+
+    /**
+     * if point annotation, do nothing
+     *
+     * if range:
+     * making sure they are not the same, and also that 'end' is after 'start'
+     */
+
+    const start = moment(startTime)
+    const end = moment(endTime)
+
+    if (end.isSame(start)) {
+      return {
+        isValid: false,
+        message: TIMES_ARE_SAME_MESSAGE,
+      }
+    }
```

Comment:
product question: should we give the user a choice to make a point annotation here? @russorat @daviddkkim

Reply:
Would allowing them to make a point annotation when start and end time is the same cause any entanglement in the way annotation is implemented? 

If it does, I don't know if it's worth the refactor because we are giving them a clear message on what is causing the error and the user can correct it.

Reply:
it wouldn't cause a problem (leaving the times the same, they *are* the same)
but the ui tells the difference between point and range because point has the same times.  
so it would make the ui, coding and usability much more complicated.
(nontrivial refactor)

Reply:
this is fine as is.